flink-runtime-web/web-dashboard/app/partials/jobs/job.jade

```diff
@@ -50,19 +50,15 @@ nav.navbar.navbar-default.navbar-fixed-top.navbar-main(ng-if="job")
 nav.navbar.navbar-default.navbar-fixed-top.navbar-main-additional(ng-if="job")
   ul.nav.nav-tabs
     li(ui-sref-active='active')
-      a(ui-sref=".plan") Plan
+      a(ui-sref=".plan") Overview
 
-    //- li(ui-sref-active='active' ng-if="job['end-time'] > -1")
     li(ui-sref-active='active')
       a(ui-sref=".timeline") Timeline
 
     li(ui-sref-active='active')
       a(ui-sref=".exceptions") Exceptions
 
     li(ui-sref-active='active')
-      a(ui-sref=".properties") Properties
-
-    li(ui-sref-active='active')
       a(ui-sref=".config") Configuration
```

Comment:
I think it was originally meant to show properties from the batch optimizer.

This is how it looked:
![image](https://cloud.githubusercontent.com/assets/89049/20007707/ddb8c3b4-a29d-11e6-9ab5-53f1b48c74c6.png)

We can add it again once the page is showing some meaningful information.